docs/pages/reference/cli.mdx

```diff
@@ -1096,6 +1096,29 @@ $ tsh request drop [<request-id>...]
   your certificate will lose elevated roles and/or resource restrictions.
   If no request ID is given, the default is to drop all Access Requests.
 
+### tsh device enroll
+
+Enroll the current device as a trusted device.
+
+Requires a device enrollment token created via `tctl devices enroll`.
+
+```code
+$ tsh device enroll --token=TOKEN
+```
+
+#### Flags
+
+| Name | Default Value(s) | Allowed Value(s) | Description |
+| - | - | - | - |
+| `--token` | none | String | Device enrollment token |
+
+#### Examples
+
+```code
+$ tsh device enroll --token=f3oJYqWfO6XG1ZCIF+faz8HIpDye54LsRFObftGKVVU
+# Device "C00AA3BBBB6C"/macOS enrolled
```

Comment:
Another spot to replace with an obviously fake token. Maybe we should also create a dummy serial number var as well?

```suggestion
$ tsh device enroll --token=f3oJYqWfO6XG1ZCIF+faz8HIpDye54LsRFObftGKVVU
Device "C00AA3BBBB6C"/macOS enrolled
```

Reply:
That serial number is a dummy, but let me know if you'd rather make it more obvious.